sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/sources/RateStreamProviderSuite.scala

```diff
@@ -78,6 +78,22 @@ class RateStreamProviderSuite extends StreamTest {
     )
   }
 
+  test("stream table API support") {
```

Comment:
I don't think this test should be here. The origin goal of the test is to verify reading from temp view via `readStream.table`, right? This should be placed in DataStreamTableAPISuite, and probably don't need to be related to the rate stream.

Reply:
Yeah, it's a leaking prototype test, will delete and use another way to test the temp view.